daemon/container_operations_unix.go

```diff
@@ -419,50 +419,20 @@ func serviceDiscoveryOnDefaultNetwork() bool {
 
 func buildSandboxPlatformOptions(container *container.Container, cfg *config.Config, sboxOptions *[]libnetwork.SandboxOption) error {
 	var err error
-	var originResolvConfPath string
 
-	// Set the correct paths for /etc/hosts and /etc/resolv.conf, based on the
-	// networking-mode of the container. Note that containers with "container"
-	// networking are already handled in "initializeNetworking()" before we reach
-	// this function, so do not have to be accounted for here.
-	switch {
-	case container.HostConfig.NetworkMode.IsHost():
-		// In host-mode networking, the container does not have its own networking
-		// namespace, so both `/etc/hosts` and `/etc/resolv.conf` should be the same
-		// as on the host itself. The container gets a copy of these files.
+	// In host-mode networking, the container does not have its own networking
+	// namespace, so `/etc/hosts` should be the same as on the host itself. Setting
+	// OptionOriginHostsPath means the container will get a copy of the host's file.
```

Comment:
[The host's hosts file?](https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo)
```suggestion
	// OptionOriginHostsPath means the container will get a copy of `/etc/hosts` from the host filesystem.
```

Reply:
Done.